tests/load/pipeline/test_restore_state.py

```diff
@@ -65,9 +65,10 @@ def test_restore_state_utils(destination_config: DestinationTestConfiguration) -
     with p.destination_client(p.default_schema.name) as job_client:  # type: ignore[assignment]
         with pytest.raises(DestinationUndefinedEntity):
             load_pipeline_state_from_destination(p.pipeline_name, job_client)
-        # sync the schema
-        p.sync_schema()
-        # check if schema exists
+    # sync the schema
+    p.sync_schema()
+    # check if schema exists
```

Comment:
Local qdrant uses flock and can't handle more than one instance of client, why I moved all the pipeline operations out of `with destination_client()` scope